docs/content/java-support.md

```diff
@@ -0,0 +1,20 @@
+## Java Language Support
+
+From version 0.0.32 we provide [VS Code Language support for java](https://marketplace.visualstudio.com/items?itemName=redhat.java) and [VS Code Debugger for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-debug) as dependencies.
```

Comment:
WDYT, use the name as in the marketplace: 
- VS Code Language support for java -> Language Support for Java(TM) by Red Hat
- VS Code Debugger for Java -> Debugger for Java